### DISCUSSION TITLE
如下图，调试时，可以像vscode一样，调试完的变量，可以看到其存在的信息

### DISCUSSION THREAD

**Discussion by wxzvim** [ayamir/nvimdots#882] [Q&A] [answered]
![image](https://github.com/ayamir/nvimdots/assets/110797126/e3c0312b-43c0-4456-a666-b9fffdfa58bc)

**Comment by Jint-lzxy** [comment, marked as answer]
Wait for #877 to merge. After that you may use `K` for this purpose.

**Reply by wxzvim** [reply]
OK
